src/node/internal/crypto.d.ts

```diff
@@ -3,3 +3,4 @@
 //     https://opensource.org/licenses/Apache-2.0
 
 export function getRandomInt(min: number, max: number): number;
+export function checkPrimeSync(candidate: ArrayBufferView, num_checks: number): boolean;
```

Comment:
```suggestion
export function checkPrimeSync(candidate: ArrayBuffer|SharedArrayBuffer|ArrayBufferView, num_checks: number): boolean;
```

Reply:
In the TypeScript implementation I'm currently passing `candidate as ArrayBufferView` – should I keep that as-is or get rid of the cast and adopt this change?

The other suggestions look useful.

Reply:
Nvm, `kj::Array<kj::byte>` actually is a view based on the JSG readme

Reply:
`kj::Array<kj::byte>` also accepts an `ArrayBuffer` and `SharedArrayBuffer`